feature/dynamodb/attributevalue/empty_collections_test.go

```diff
@@ -913,7 +915,7 @@ func TestEmptyCollectionsSpecialCases(t *testing.T) {
 	if err != nil {
 		t.Fatalf("expected no err got %v", err)
 	}
-	if diff := cmp.Diff(expectedEncode, actualEncode); len(diff) != 0 {
+	if diff := cmp.Diff(expectedEncode, actualEncode, cmpopts.IgnoreTypes(smithydocument.NoSerde{})); len(diff) != 0 {
```

Comment:
Will having the embedded `smithydocument.NoSerde{}` in the SDK types cause similar issues with customer applicaiton's doing compares on SDK generated types?

Reply:
Discussed offline, it was determined that this was likely the best course of action compared to other alternatives.